substrate/frame/src/lib.rs

```diff
@@ -159,6 +295,10 @@ pub mod testing_prelude {
 	};
 
 	pub use frame_system::{self, mocking::*};
+
+	#[deprecated(note = "Use `frame::testing_prelude::TestExternalities` instead.")]
```

Comment:
Should also mention when this will finally be removed. Otherwise people never switch 😆

Reply:
Good point! Typically we allocate 6-9 months? So you can add a note that it will be removed June 2025.